zappa/cli.py

```diff
@@ -450,7 +450,9 @@ def create_package(self):
                 self.lambda_name,
                 handler_file=handler_file,
                 use_precompiled_packages=self.zappa_settings[self.api_stage].get('use_precompiled_packages', True),
-                exclude=self.zappa_settings[self.api_stage].get('exclude', [])
+                exclude=self.zappa_settings[self.api_stage].get('exclude', []),
+                exclude_conda_packages = self.zappa_settings[self.api_stage].get('exclude_conda_packages',
+                    ['pip','python','readline','sqlite','wheel', 'boto3', 'botocore'])
```

Comment:
I believe openssl and tk (among other things) are included with miniconda as well. You probably won't want them in your lambda function either.